rllib/algorithms/marwil/tests/test_marwil.py

```diff
@@ -29,7 +29,7 @@
 class TestMARWIL(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        ray.init()
+        ray.init(local_mode=True)
```

Comment:
nit: remove

Reply:
Good catch! Might even explain the noisy output from ray.data